### PR TITLE
feat(aens): implement aensExtendTtl function. Refactor aensUpdate

### PR DESCRIPTION
**Pull request by nduchak**
aensUpdate now accept array of pointers

BREAKING CHANGE: Change aensUpdate method arguments

#865

### CODE REVIEW

**Comment by mradkov** [comment]
Great! Will this be included in v7.0.0 ?

**Comment by nduchak** [comment]
Yes, it will

**Comment by nduchak** [comment]
Resolve #865

File: docs/api/ae/aens.md [hunk 1]
```diff
@@ -41,45 +41,94 @@ Aens provides name-system related methods atop
 
 <a id="exp_module_@aeternity/aepp-sdk/es/ae/aens--update"></a>
 
-### .update(nameId, target, options) ⇒ `Object` ⏏
-Update an aens entry
+### .update(name, pointers, [options]) ⇒ `Promise.&lt;Object&gt;` ⏏
+Update an name
```
Comment (mradkov) [inline docs/api/ae/aens.md]: Just a small typo in the documentation: 
 > Update **a** name

File: es/ae/aens.js [hunk 2]
```diff
@@ -27,49 +27,89 @@
...
+    nameId: produceNameId(name),
     accountId: await this.address(opt)
   }))
 
   return this.send(nameRevokeTx, opt)
 }
 
 /**
- * Update an aens entry
+ * Update an name
```
Comment (mradkov) [inline es/ae/aens.js]: > Update **a** name

**Comment by davidyuk** [comment]
it seems that I can't find where `aensExtendTtl` is defined

**Pull request merged**